lib/importexport/helpers.js

```diff
@@ -123,7 +123,16 @@ var _importZip = function(filePath, cb, retries){
     if (!zipEntries || zipEntries.length ===0){
       return cb('No zip entries found');
     }
-    for (var i=0; i<zipEntries.length; i++){
+
+    for (var i = 0; i < zipEntries.length; i++){
+      if (zipEntries[i].isDirectory && zipEntries[i].entryName.indexOf('__MACOSX') > -1) {
+        zip.deleteFile(zipEntries[i])
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)